sui/cli-utils.js

```diff
@@ -51,7 +51,16 @@ const addExtendedOptions = (program, options = {}) => {
     return program;
 };
 
+const addBaseOptionsToCommands = (program, options = {}) => {
```

Comment:
```suggestion
// `options` is a method such as `addBaseOptions`
const addOptionsToCommands = (program, options) => {
```
make it more generic